wire/core/WireArray.php

```diff
@@ -1679,13 +1679,20 @@ public function getIterator() {
 	 * $qty = count($items); 
 	 * ~~~~~
 	 * 
+	 * You can also pass a selector:
+	 * 
+	 * ~~~~~
+	 * $qty = $items->count("foo=bar");
+	 * ~~~~~
+	 * 
 	 * #pw-group-retrieval
 	 * 
 	 * @return int
```

Comment:
This DocBlock should have a param definition, perhaps something along the lines of `@param string|null $selector Optional selector string to match items against.`